src/components/ItemTable/RequestButtons.tsx

```diff
@@ -20,7 +20,7 @@ const RequestButtons = ({ item }: RequestButtonsProps) => {
         <ExternalLink
           href={item.aeonUrl}
           type={!item.isAvailable ? "buttonDisabled" : "buttonSecondary"}
-          aria-label={`Request Appointment, ${item.bibTitle}`}
+          aria-label={`Request Appointment, ${item.requestTitle}`}
```

Comment:
This works well but I'd just rename this to be more specific. This will only really be used here so perhaps it could include "aria" in the variable name. `requestTitle` makes it seem like it could be use elsewhere.